analysis/cross_partition_combiners.py

```diff
@@ -156,17 +162,20 @@ def _multiply_float_dataclasses_field(dataclass, factor: float) -> None:
             _multiply_float_dataclasses_field(value, factor)
 
 
-def _per_partition_to_cross_partition_utility(
+def _per_partition_to_cross_partition_metrics(
```

Comment:
nit: This returns the UtilityReport, AFAIU the last step in the process. If so, the name and the doctring doesn't really match that.